app/custom/templates/not-found/_style.scss

```diff
@@ -0,0 +1,3 @@
+.t__not-found {
```

Comment:
We should leave these files, as they help guide people into properly structuring their app.